hexrd/ui/import_data_panel.py

```diff
@@ -99,8 +99,8 @@ def get_instrument_defaults(self):
             else:
                 try:
                     with h5py.File(self.config_file, 'r') as f:
-                        instr = create_hedm_instrument()
-                        instr.unwrap_h5_to_dict(f, self.defaults)
+                        instr = HEDMInstrument(f)
```

Comment:
I'm not 100% sure of everything that is going on here, but you may need to pass in the `tilt_calibration_mapping` similar to what is done [here](https://github.com/HEXRD/hexrdgui/blob/79b0b0cf568392fac413a63a0500ddf8f097c67b/hexrd/ui/hexrd_config.py#L496) via `HexrdConfig().rotation_matrix_euler()`.